### DISCUSSION TITLE
limit max sample rate

### DISCUSSION THREAD

**Discussion by 7enso** [vincentneo/LosslessSwitcher#39] [Ideas]
My audio device is capable of playing at 192 kHz or more, my speakers are as well (digital in). Due to cable length issues the SPDIF signals get distorted above 96 kHz. So if the sample rate switches above 96 kHz I get distorted audio.

Therefore it would be VERY useful to limit the maximum sample rate to a specific value - at least for me.

Implementation of that feature would be great! Thanks for the great work already :)

**Comment by vincentneo** [comment]
I've recalled seeing a similar situation, but with some USB DACs, where it's reporting higher than supported sample rate, and refusing to playback when those sample rates are selected.

A feature to consider adding indeed. Thanks for suggesting!
